avocado/core/sysinfo.py

```diff
@@ -411,6 +411,20 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
             log.debug('File %s does not exist.', commands_file)
             self.commands = []
 
+        fail_commands_file = settings.get_value('sysinfo.collectibles',
```

Comment:
We are going to deprecate .settings.py soon. Please use future.settings here.

Reply:
But we still were using `settings` in the other places in sysinfo, so the confusion.
Will use `future.settings`.

Reply:
@beraldoleal Do you want to do this now or separately ? Reason is, there are other places where we have to do this change and might require more testing

Reply:
@narasimhan-v sorry for the late reply. I submitted a few PRs migrating (and I'm still doing this) a few options to use the new API.

Looks like we have already all options related to sysinfo using the new API because those PRs were already merged. So yes, I think that we can change this PR to not use settings.get_value(). You should use `config.get()` if your method has access to the config or `future_settings.as_dict().get()` if not.

Reply:
@beraldoleal Thanks, have raised a v3 (https://github.com/avocado-framework/avocado/pull/3923), please have a look.